app/scripts/modules/dcos/pipeline/stages/disableCluster/disableClusterExecutionDetails.controller.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+import { EXECUTION_DETAILS_SECTION_SERVICE } from '@spinnaker/core';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.disableCluster.dcos.executionDetails.controller', [
+  EXECUTION_DETAILS_SECTION_SERVICE,
+])
+  .controller('dcosDisableClusterExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService) {
+
+    $scope.configSections = ['disableClusterConfig', 'taskStatus'];
+
+    let initialized = () => {
+      $scope.detailsSection = $stateParams.details;
+    };
+
+    let initialize = () => executionDetailsSectionService.synchronizeSection($scope.configSections, initialized);
+
+    initialize();
+
+    $scope.$on('$stateChangeSuccess', initialize);
+
+  });
```

Comment:
Should probably just use the `BaseExecutionDetailsCtrl` instead of creating this.